packages/waku/src/router/create-pages.ts

```diff
@@ -135,14 +136,14 @@ export type CreateLayout = <Path extends string>(
   layout:
     | {
         render: 'dynamic';
-        path: PathWithoutSlug<Path>;
+        path: Path;
```

Comment:
I'm not sure about this change, but count on you.

Reply:
without this change there is a type error if we attempt to define a layout for `/foo/[id]` which I think would be valid.

the fs router equivalent would be `pages/foo/[id]/_layout.tsx`